todo.org

```diff
@@ -81,3 +77,9 @@ Seems a bit heavy handed but its working for me so far.
 
 ** TODO is disabling color still required?
    in the readme it mentions that color should be turned off. in my usage I haven't run into this problem at all. perhaps no longer true?
+** TODO nice startup
+There's some project detection but that's becoming less and less useful as time goes on. Shadow, lein, deps.edn can all easily be mixed in the same project. And then lumo, planck, or bb scripts could live side by side. Rather than trying to guess the project type, I think i'd like to mimic geiser's style of handling multiple scheme backends. Perhaps ~m-x inf-clojure-run-planck~ and similar could help out.
+
+Some considerations:
+- is this path aware? IE, don't show an option to run planck, lumo, etc, if they aren't visible or installed?
+- should it have a rebuild function so that user registered implementations can show up in the ~m-x~ menu as well?
```

Comment:
If we go for the classic defcustom as list approach and maybe a register function like `lsp` imho yes